src/jsonschema_to_typeddict/converter.py

```diff
@@ -127,6 +136,12 @@ def _convert_object_entry(entry_name: str, entry_value: dict) -> CodeResult:
     return CodeResult(block_result, type_name)
 
 
+def _convert_enum_entry(entry_name: str, entry_value: dict) -> CodeResult:
+    values = [repr(value) for value in entry_value["enum"]]
+    inline = _snake_case_to_pascal_case(entry_name)
+    return CodeResult(f"{inline} = typ.Literal[\n    {',\n    '.join(values)}\n]\n", inline)
```

Comment:
Broke Python < 3.12

Reply:
ugh, are we not using typing extensions? damn

Reply:
It's actually an issue with the f-string

Reply:
> It's actually an issue with the f-string

oh LMAO ok that's easily fixed phew